ci/scripts/get-secret.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/zsh
```

Comment:
we assume that Z Shell is always installed? Do these scripts use extensions available only in zsh?